mythtv/libs/libmythtv/mythplayer.cpp

```diff
@@ -2201,12 +2215,15 @@ bool MythPlayer::PrebufferEnoughFrames(int min_buffers)
     if (!videoOutput)
         return false;
 
-    if (!(min_buffers ? (videoOutput->ValidVideoFrames() >= min_buffers) :
```

Comment:
This code change looks unrelated to the problem it is fixing.

Reply:
Change here is adding the log statement below. Hitting the "SetBuffering" call is a primary indicator of the problem I'm addressing with these changes. But yeah, it can go away if preferred.

Reply:
Ah right, that makes sense. It can stay :)